code/modules/events/abductor.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/abductor
+	name = "Abductors"
+	typepath = /datum/round_event/abductor
+	weight = 5
+	max_occurrences = 1
+
+	earliest_start = 180000 // 30 min
+
+	gamemode_blacklist = list("nuclear","wizard","cult","gang","revolution","abduction","blob")
```

Comment:
Abductors have some windup time so i didn't want to add them to fast-paced rounds (nuke,wizard,rev) but yeah they could probably work in the other ones.